### DISCUSSION TITLE
How to remove audio using the filter plugin?

### DISCUSSION THREAD

**Discussion by tremzo** [ant-media/Ant-Media-Server#5725] [FAQs]
I am trying to use the filter plugin to create a new stream where the audio is removed. 

This is the API request I am using:

```
curl -i -X POST -H "Accept: Application/json" -H "Content-Type: application/json" "http://localhost:5080/LiveApp/rest/v2/filters/create" -d '{"filterId":"filter1","inputStreams":["stream1"],"outputStreams":["stream2"],"videoFilter":"[in0]acopy[out0]","audioFilter":"[in0]acopy[out0]","videoEnabled":"true","audioEnabled":"false"}'
```

However, I get an error:

`{"success":false,"message":"Video filter graph can not be initiated:[in0]acopy[out0]","dataId":null,"errorId":0}%  `

I just copied the request from the blog post here https://antmedia.io/discover-the-filter-plugin/ that is used to flip the video using `vflip` but I replaced `vflip` with `acopy`.

**Comment by USAMAWIZARD** [comment]
let me check it and get back to you soon

**Reply by tremzo** [reply]
@USAMAWIZARD Actually, I tried it again on a new installation and it worked. I used the sample request on this post [
](https://antmedia.io/discover-the-filter-plugin/) that flips the video. My problem is how to not flip video. I tried chaning `vflip` to `acopy` but it errors. 

This causes an error

`"videoFilter":"[in0]acopy[out0]"`

So what I tried is this but errors:

```
curl -i -X POST -H "Accept: Application/json" -H "Content-Type: application/json" "http://localhost:5080/WebRTCAppEE/rest/v2/filters/create" -d '{"filterId":"filter1","inputStreams":["stream1"],"outputStreams":["test"],"videoFilter":"[in0]acopy[out0]","audioFilter":"[in0]acopy[out0]","videoEnabled":"true","audioEnabled":"false"}'
```
